apps/src/StudioApp.js

```diff
@@ -251,6 +251,15 @@ StudioApp.prototype.init = function(config) {
     config = {};
   }
 
+  if (config.isLegacyShare && config.controllerAction == 'show') {
```

Comment:
We could keep it specific to sharing and add a config.isLegacyShareShow or something like that:

if (config.isLegacyShare && config.isLegacyShareShow )

Thoughts?